flow/envs/bottleneck_env.py

```diff
@@ -484,14 +454,10 @@ def get_state(self):
             else:
                 id_counter += 1
             num_lanes = MAX_LANES * self.scaling
-            headway = np.asarray([1000
-                                  for _ in range(num_lanes)]) / headway_scale
-            tailway = np.asarray([1000
-                                  for _ in range(num_lanes)]) / headway_scale
-            vel_in_front = np.asarray([0 for _ in range(num_lanes)
-                                       ]) / self.max_speed
-            vel_behind = np.asarray([0 for _ in range(num_lanes)
-                                     ]) / self.max_speed
+            headway = np.asarray([1000] * num_lanes) / headway_scale
```

Comment:
I always get nervous using `[...] * ...` because it creates copies of the same variable instead of generating new ones in memory. Maybe switch to:
```
headway = np.ones(num_lanes) * 1000 / headway_scale
```
same for the three variables below.

Reply:
There's no problem when you use `[...] * ...` with immutable types like `int` since you can never modify their value so it doesn't matter that they're stored at the same location ([link](https://stackoverflow.com/a/24557558/11584932), [link](https://stackoverflow.com/a/983727/11584932))

Reply:
awesome, I'm convinced